### DISCUSSION TITLE
Full screen support for phantom extension

### DISCUSSION THREAD

**Discussion by GHkrishna** [phantom/docs#92] [General]
There seems to be no full-screen interface support for the browser-based extension in order to efficiently navigate and perform complex operations.

While performing operations such as wanting to extract keys or other where there is a constant need for the wallet interface to be open in a new tab. The extension wallet seems to be closing as we change tabs or windows(which is understandable). But it t becomes quite frustrating to constantly open the extension, navigate to the desired settings and sub-settings, and use a password to reveal the passphrase (in case of viewing the passphrase).

This feature would be something similar to the expand view functionality in metamsk

I'm sure there might be other workflows where an expanded view will be very beneficial. If there are any PRs, issues, or discussions to that please guide me by providing a link to them.

If there's already expanded view support(at least I'm unable to find one), please let me know.
 
Bowser: Chomium Version 116.0.5845.141 (Official Build) (64-bit)

**Comment by adamdelphantom** [comment]
Hello @GHkrishna,

Thank you for writing in!

Please consider posting this on our feature request board [here](https://phantom.canny.io). We actively source this board for future feature development. It also allows others to vote and weigh in on the request.

**Reply by GHkrishna** [reply]
Thankyou, will add it there.
